libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -190,7 +190,8 @@ void AP_GPS_Backend::send_mavlink_gps_rtk(mavlink_channel_t chan)
 {
     const uint8_t instance = state.instance;
     // send status
-    if (instance == 0) {
+    switch (instance) {
+        case 0:
         mavlink_msg_gps_rtk_send(chan,
```

Comment:
Not to be the style obsessed person but either everything (including the break) needs to be tabbed in one more so that it's in from the case, or the case needs to be untabbed one. http://ardupilot.org/dev/docs/style-guide.html#case-statements.